presto-main/src/main/java/com/facebook/presto/operator/scalar/UrlFunctions.java

```diff
@@ -129,6 +132,42 @@ public static Slice urlExtractParameter(@SqlType(StandardTypes.VARCHAR) Slice ur
         return null;
     }
 
+    @Nullable
+    @Description("encode url")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice urlEncode(@SqlType(StandardTypes.VARCHAR) Slice url)
```

Comment:
The argument is not really a url, but a value to be included in a url query parameter. Name it `value` instead.